spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/server/db/migration/AbstractSmokeTest.java

```diff
@@ -84,7 +84,7 @@ static void databaseProperties(DynamicPropertyRegistry registry) {
 
 	@Test
 	public void testTaskCreation() {
-		logger.info("testTaskCreation:started:{}", getClass().getSimpleName());
+		assertThat(taskExplorer.getTaskExecutionCount()).isEqualTo(0);
```

Comment:
This was added to assert up front that the world is clean otherwise it can waste quite a bit of time if this regresses to understand what is going on. Basically this is a post condition. 

This test is really just trying to make sure that 2 new execs were created and the count has increased by 2.

Another modification which is more flexible is to grab teh original coutn and make sure it is increased by 2. The benefit of making sure it is zero is that it signals that there is some other bad actor test not cleaning up after itself.